packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -605,3 +616,27 @@ function updateHandleCursor(handles: HTMLElement[], angleRad: number) {
         handle.style.cursor = `${rotateHandles(handle, angleRad)}-resize`;
     });
 }
+
+/**
+ * Check if the current image was resized by the user
+ * @param image the current image
+ * @returns if the user resized the image, returns true, otherwise, returns false
+ */
+function checkIfImageWasResized(image: HTMLImageElement): boolean {
+    const { width, height, style } = image;
+    if (
+        style.maxWidth === 'initial' &&
+        isFixedNumberValue(style.height) &&
+        isFixedNumberValue(style.width) &&
+        width &&
+        height
```

Comment:
1. we also need to call isFixedNumberValue for width and height
2. These 4 should be connected using "or". Any of them exists, we need to treat it as resized